### DISCUSSION TITLE
Form `target` usage

### DISCUSSION THREAD

**Discussion by freshollie** [remix-run/remix#8017] [Q&A] [answered]
Does Remix support/plan to support the `target` prop on `Form`?

I want to submit a form to a new tab (in some cases). This is supported by the native html form: https://www.w3schools.com/tags/att_form_target.asp

If I pass this prop to the Remix form at the moment the data is still submitted via JS and a new tab is not opened. If I use the native form I get the desired behaviour.

**Comment by sergiodxa** [comment, marked as answer]
You can use target with reloadDocument, it will be similar to using a form tag directly but with the benefit that Form correctly points the action to the current route action.

**Reply by freshollie** [reply]
Nice! I think this is what I was missing :)
